src/commands/helpAndFeedback/reportIssue.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { commands, env, Uri } from "vscode";
+import { IActionContext, IAzureQuickPickItem } from "vscode-azureextensionui";
+import { AzExtWrapper, getInstalledExtensionPicks } from "../../AzExtWrapper";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../utils/localize";
+
+export async function reportIssue(context: IActionContext): Promise<void> {
+    const picks: IAzureQuickPickItem<AzExtWrapper | undefined>[] = getInstalledExtensionPicks();
+    picks.push({
+        label: localize('other', 'Other'),
+        data: undefined
+    });
+
+    const placeHolder: string = localize('selectExtension', 'Select the Azure extension you want to report an issue on');
+    const azExtension: AzExtWrapper | undefined = (await ext.ui.showQuickPick(picks, { placeHolder, suppressPersistence: true })).data;
```

Comment:
Do we want to turn off persistence?  I could see it being useful if you always report issues on the same extension.

Reply:
My thinking on this was - I don't like persistence for seldom-used quick picks because the thing I did a week or two ago probably isn't what I want to do now. If people are _not_ using this seldomly, I feel like we have bigger problems 😝  I also feel like persistence is more important for long wizards and this is pretty simple